packages/worker/src/utils.ts

```diff
@@ -1,8 +1,5 @@
-export const sleep = (ms: number): Promise<void> => new Promise((resolve) => setTimeout(resolve, ms));
-export const identity = <T>(x: T): T => x;
```

Comment:
`identity` was unused. `sleep` was only used in a test file so I moved it there.